### PR TITLE
Docs: vault with an expired license

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

Explain what happens when using Vault with an expired enterprise license.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by sakjur — DISMISSED** [review]

File: docs/sources/enterprise/license-expiration.md [hunk 1]
```diff
@@ -64,4 +64,4 @@ All the usage insights features are turned off, meaning that you won't be able t
 
 ### Vault integration
 
-Vault integration is not affected by an expired license.
+Grafana will fail to boot if the Vault integration is used with an expired license. Update the license or disable the Vault integration for Grafana to boot again.
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/license-expiration.md:67]: Change the second sentence to: 
You must update the expired license or disable the Vault integration for Grafana to boot up.

**Review by achatterjee-grafana — COMMENTED** [review]
Requested a change.

File: docs/sources/enterprise/license-expiration.md (hunk 1 above)
Comment (xlson) [reply docs/sources/enterprise/license-expiration.md:67]: Thanks for the feedback. It turns out I was wrong about how it works so I'll close this PR.

**Comment by xlson** [comment]
I misremembered how it works.

**Pull request closed without merging**